src/components/Courses/Courses.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+
+import { useState } from 'react';
+
+import Button from 'src/common/Button/Button';
+
+import { getCourseById } from 'src/helpers/getCourseById';
+
+import CourseCard from './components/CourseCard/CourseCard';
+import SearchBar from './components/SearchBar/SearchBar';
+import EmptyCourseList from '../EmptyCourseList/EmptyCourseList';
+import CourseInfo from '../CourseInfo/CourseInfo';
+
+import { ADD_COURSE_BTN } from 'src/constants';
+
+interface coursesProp {
+	data: object[];
+}
+
+export default function Courses(props: coursesProp) {
+	const { data } = props;
+
+	const [courses, setCourses] = useState(data);
+	const [showInfo, setShowInfo] = useState(false);
+	const [courseId, setCourseId] = useState('');
+	const components =
```

Comment:
Let's try to put this JSX directly into the return statement, I think it should improve readability of this component